policyengine_us/parameters/gov/states/nc/ncdhhs/scca/entry/income_rate_by_child_age.yaml

```
@@ -0,0 +1,20 @@
+description: North Carolina Subsidized Child Care Assistance program's maximum initial income limits differ depending on the age of the child
+metadata:
+  type: single_amount
+  threshold_unit: year
+  amount_unit: /1
+  label: North Carolina Subsidized Child Care Assistance program income limits by child age
+  reference:
+    - title: Subsidized Child Care Assistance Program Policy Manual
+      href: https://policies.ncdhhs.gov/wp-content/uploads/chapter-8-parental-fees-7.pdf
```

add page number (9)

---

Specifically if it helps
```suggestion
      href: https://policies.ncdhhs.gov/wp-content/uploads/chapter-8-parental-fees-7.pdf#page=9
```

---

Thanks both.